### ISSUE TITLE
\change Staff is not supported

### ISSUE THREAD

**Issue by uliska** [wbsoft/python-ly#28]
This is not a complaint but just to mark it as an enhancement request.

`\change Staff` is not supported yet (and actually I don't know how this can be encoded in MusicXML).

**Comment by PeterBjuhr** [comment]
Much of the basic support is already in place here. But there are things to consider.

The following example can be exported and reimported:

````lilypond
\score {
  \new PianoStaff <<
    \new Staff = "1" {
      \clef treble
      c'8 \change Staff="2" a
      \change Staff="1" d' e'
      \change Staff="2" a
      \change Staff="1"
      d'
      \change Staff="2"
      a e |
      \change Staff="1"
      R1
    }
    \new Staff = "2" {
      \clef bass
      s1 |
      c1
    }
  >>
}
````

(If we remove the second bar the music can't be reimported, but I think that has to be interpreted as a musicxml2ly issue.)

The thing to consider is that if we change from `PianoStaff` to `StaffGroup` (as in the Ferneyhough example) it doesn't work anymore because the staffs or no longer in the same part. (I did mention this briefly in #24.)

The only possible solution I can think of at the moment is to reshape the structure when `\change Staff` occurs in a `StaffGroup`. Or is there another way to see beforehand that this should be the same part?

So to sum up: in LilyPond staffchanges can take please between any staffs, but in MusicXML they can only take place in a part with multiple predefined staffs.
